content/en/aws/docdb/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "DocumentDB (DocDB)"
+linkTitle: "DocumentDB (DocDB)"
+categories: ["LocalStack Pro"]
+description: >
+  DocumentDB (DocDB)
+---
+
+LocalStack supports a basic version of [Amazon DocumentDB](https://aws.amazon.com/documentdb/) for testing.
+
+When loading the DocumentDB service, LocalStack will create a MongoDB container to handle DocumentDB storage. By default, this instance will listen on a port in the 33000-34000 port range and will instantiate a master user named 'test' without a password. The `mongosh` client may be used to log in to and interact with the instance.
```

Comment:
Hi Localstack Team,

Please verify these defaults. This is just what I noticed from using the service myself.

Reply:
Hi @snhardin, 

thanks for generating the docs for DocumentDB 😄 👍 
As you already pointed out, the current implementation uses MongoDB and provides a very basic functionality. 

**Regarding the port-range**: LocalStack starts a the mongo-db in a separate docker-container and adds a port-mapping, so that the mongo-db can be accessed from localhost. To find such free port, LocalStack tries to find a free port on the machine (more or less randomly) - so there is no pre-defined port range.

**Regarding the database and defaults:** currently, we use the default configuration of the latest [mongo docker image](https://hub.docker.com/_/mongo). It automatically creates a database named `test` if nothing else is defined (which currently is the case).